blockindex/bloomfilterindexer.go

```diff
@@ -128,7 +131,10 @@ func (bfx *bloomfilterIndexer) DeleteTipBlock(blk *block.Block) (err error) {
 		return err
 	}
 	if (height-1)%bfx.rangeSize == 0 {
-		bfx.curRangeBloomfilter = nil
+		bfx.curRangeBloomfilter, err = bloom.NewBloomFilter(2048, 3)
+		if err != nil {
+			return err
+		}
```

Comment:
we need to delete the corresponding value in db.

Reply:
In line 130, at delete(), we are deleting the value at height in db, right?